sequencer/src/main/java/net/consensys/linea/sequencer/txselection/selectors/MaxBlockGasTransactionSelector.java

```diff
@@ -27,13 +27,29 @@
 import org.hyperledger.besu.plugin.services.txselection.PluginTransactionSelector;
 import org.hyperledger.besu.plugin.services.txselection.TransactionEvaluationContext;
 
+/**
+ * This class implements TransactionSelector and provides a specific implementation for evaluating
+ * if the cumulative gas used by the block, including the current pending transaction, is below the
+ * user configured max amount, if not the transaction is not selected. This means that the user can
+ * configure a max gas per block that is below the limit defined by the protocol.
+ */
 @Slf4j
 @RequiredArgsConstructor
 public class MaxBlockGasTransactionSelector implements PluginTransactionSelector {
 
   private final long maxGasPerBlock;
   private long cumulativeBlockGasUsed;
 
+  /**
+   * Evaluates a transaction post-processing. Checks if adding the gas used of the transaction, to
+   * the cumulative gas used of the block till now, is below the user configured max gas used per
```

Comment:
Does user here mean, the node that is going to create the block, basically the sequencer in Linea ?

Reply:
it means who is running the sequencer, I was not sure about _user_, do you think _operator_ is better in this context?